bin/templates/cordova/Api.js

```diff
@@ -349,13 +354,24 @@ Api.prototype.run = function(runOptions) {
  */
 Api.prototype.clean = function(cleanOptions) {
     var self = this;
+    try {
```

Comment:
I think instead of doing this `try catch` in product code we could just mock `Api.prototype.clean()` in tests. Please see my attempt at doing so:
https://github.com/alsorokin/cordova-android/commit/008f46a1f3e807ee0ec1b576cd1a5d84e779d838